pagopa/pn-frontend
# Feature/PN-10858 Sender Dashboard Components 2/2

## Short description
This PR completes the work of development and integration of every component into the Sender Dashboard

## List of changes proposed in this pull request
- add translations to statistics.json for the newly developed components
- change the AggregateStatistics component to accept more optional configuration props
- add custom formatter to statistics components' tooltips
- change color palette to every statistics component to follow the new specifications
- add DigitalErrorsDetailsStatistics component to show technichal errors by error type
- add DigitalMeanTimeStatistics component to show the mean number of days for notifications to reach the states of delivered, viewed and refined
- add DigitalStateStatistics component to show the number of digital notifications by outcome
- add EmptyStatistics component to be displayed when the number of notifications sent is not enough to receive statistics data from the server (empty state)
- add IllusStatistics component in commons to draw the statistics empty state svg using the Illustration component

## How to test
Prerequisite: IS_STATISTICS_ENABLED env variable should be created and set to true for pa-webapp.

- To test the empty state go to api/statistics/statistics.mock.ts at line 10 and explicitely select the 5th (index 4) element in the mocks array.
- To test the other functionalities select one of the other mocks which contain some data (0-3)
